tests/unit/lms/services/organization_usage_report_test.py

```diff
@@ -163,18 +163,20 @@ def test_usage_report(
         factories.LMSCourseApplicationInstance(
             lms_course=lms_course_child, application_instance=ai_child_org
         )
-        # Course created after the until date
-        course_created_after_date = factories.Course(
-            application_instance=ai_child_org,
-            created=until + timedelta(days=1),
+
+        # Course last used before the start date
+        factories.Course(
+            application_instance=ai_root_org,
+            updated=since - timedelta(days=2),
+            created=since - timedelta(days=2),
         )
-        lms_course_created_after_date = factories.LMSCourse(
-            h_authority_provided_id=course_created_after_date.authority_provided_id,
+
+        # Course created after the until date
+        factories.Course(
+            application_instance=ai_root_org,
+            updated=until + timedelta(days=1),
             created=until + timedelta(days=1),
         )
-        factories.LMSCourseApplicationInstance(
-            lms_course=lms_course_created_after_date, application_instance=ai_child_org
-        )
```

Comment:
I removed `LMSCourse` and `LMSCourseApplicationInstance`s here which had no effect. Only the `Course` instances matter, because those are what should be excluded from the request to h for groups annotated during the given period.